bluesky/run_engine.py

```diff
@@ -880,8 +897,10 @@ def _resume_task(self, *, init_func=None):
                 init_func()
 
             if self._task_fut is None or self._task_fut.done():
-                return
-
+                try:
+                    return self._task_fut.result()
+                except:
+                    return object()
```

Comment:
you will want to create this once at the top of this file like

```
NO_PLAN_RETURN = object()
```

so that later you can do

```
ret = RE(...) 
if ret is NO_PLAN_RETURN:
    ...
else:
   ...
```

Reply:
What scope would this be in? In this example would you have run `from bluesky import NO_PLAN_RETURN`?

Reply:
That is one option.

I could also see doing this as a instance variable on the RE (so it can have a nice repr and "know" who its RE is etc) so you could do

```
ret = RE(...)
if ret is not RE.NO_PLAN_RETURN:
 ....
```

Reply:
Should I shove the definition of `NO_PLAN_RETURN` into `bluesky/utils/__init__.py`?